buildlogger/logger_producer.go

```diff
@@ -10,6 +10,7 @@ import (
 // options.LoggerProducer for BuildloggerV3.
 type JasperLoggerOptions struct {
 	Name          string         `json:"name" bson:"name"`
+	Prefix        string         `json:"prefix" bson:"prefix"`
```

Comment:
why do we need to a separate Prefix field? Isn't it already part of the `BuildloggerV3` field (a LoggerOptions)?

Reply:
No, other loggers may have a prefix option but BuildloggerV3 does not.

Reply:
why doesn't it? It is a field in the underlying LoggerOptions, isn't it?

Reply:
I am confused, what LoggerOptions are you referring to? These are the logger options for the cedar logger in jasper.

Reply:
I meant to say: [this field](https://github.com/evergreen-ci/timber/pull/55/files#diff-fcb0d4d2dd7b07a90be06d2936af5e954cbc085aa9ad0f9390b4b98109bf492eR15) in the JasperLoggerOptions struct is of type `LoggerOptions`. Since Prefix is [being added](https://github.com/evergreen-ci/timber/pull/55/files#diff-c0f367bb77f58f344f5c68b1b496990dce399ebd90cf239ba7e1f0bfa2375ab6R104) as a field in LoggerOptions why do we need to separately add a Prefix field here too?

Reply:
Oh whoops! Now I see what you are saying.